src/auth/googleauth.ts

```diff
@@ -456,6 +458,8 @@ export class GoogleAuth {
     } else {
       (options as JWTOptions).scopes = this.scopes;
       client = new JWT(options);
+      client.defaultServicePath = this.defaultServicePath;
+      client.useJWTAccessAlways = this.useJWTAccessAlways || false;
```

Comment:
You shouldn't need to include the ` || false` here.  The `||` checks if the variable has a falsey value, which would get evaluated to `false` anyways.

Reply:
here and below